packages/dashboard/src/app/views/exploreTemplates/content/templateGridItem.js

```diff
@@ -113,20 +115,22 @@ const TemplateGridItem = forwardRef(
                   {TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS[status]}
                 </Button>
 
-                <Button
-                  size={BUTTON_SIZES.SMALL}
-                  type={BUTTON_TYPES.PRIMARY}
-                  onClick={onCreateStory}
-                  disabled={!onCreateStory}
-                  tabIndex={tabIndex}
-                  ariaLabel={sprintf(
-                    /* translators: %s: template title.*/
-                    __('Create new story from %s', 'web-stories'),
-                    title
-                  )}
-                >
-                  {__('Use template', 'web-stories')}
-                </Button>
+                {apiCallbacks?.createStoryFromTemplate && (
```

Comment:
From a readability perspective it's odd that this UI component is checking for API callbacks.

Maybe that's better handled at a different level?

For example, in `TemplateGridView` we could do something like this:

```js
const canCreateStory = Boolean(templateActions.createStoryFromTemplate);

// ...

<TemplateGridItem
              detailLink={resolveRoute(centerTargetAction)}
              onCreateStory={canCreateStory ? () => handleUseStory({ id, title }): null}

// ...
```

Then, here we just check for `onCreateStory` instead of `apiCallbacks?.createStoryFromTemplate`